pkgs/top-level/python-packages.nix

```diff
@@ -10254,7 +10254,9 @@ self: super: with self; {
 
   pykrakenapi = callPackage ../development/python-modules/pykrakenapi { };
 
-  pylance = callPackage ../development/python-modules/pylance { };
+  pylance = callPackage ../development/python-modules/pylance {
+    inherit (pkgs) protobuf;
+  };
```

Comment:
Is there a reason to make this explicit here? It doesn't seem to add anything, as protobuf is already in the pylance/default.nix set of arguments.

Reply:
We want `pkgs.protobuf`, not `python.pkgs.protobuf`.